README.md

```diff
@@ -1,3 +1,11 @@
-# Experimental Scan Prime
+# README
 
-Our first Expertimental Malware Static Analysis
+### Usage
+
+Go to working directory of project, and run `cargo run -- /path/to/pe/file`.
+The csv and parquet file will be placed in the `result` folder.
+
+### Known issues
+
+- Does not work on all PE files. currently only supportes 64-bit executables
```

Comment:
Is this a limitation of the parser? Or requires further work from this code?

Reply:
More work with the code. I'm accessing data that is not always available/consistent for all PEs (32bit vs 64bt, exe vs DLL, etc.), and not checking what version/type the file is yet.